setup.cfg

```diff
@@ -2,6 +2,6 @@
 exclude = ve,docs,nurseconnect/migrations/*
 ignore = F403, F405
 
-[pytest]
+[tool:pytest]
```

Comment:
new way of specifying parameters for pytest. They've deprecated the just using `pytest`